yt_dlp/extractor/bbc.py

```diff
@@ -1235,26 +1358,93 @@ def parse_media(media):
                         'timestamp': item_time,
                         'description': strip_or_none(item_desc),
                     })
+
             for resp in (initial_data.get('data') or {}).values():
                 name = resp.get('name')
```

Comment:
Did this (or similar) get lost?
```suggestion
            for resp in traverse_obj(initial_data, ('data', lambda _, v: v['name'])):
                name = resp['name']
```